rse-r-package.Rmd

```diff
@@ -0,0 +1,590 @@
+# R Packaging {#rse-r-package}
+
+## Questions {#rse-r-package-questions}
+
+```{r, child="questions/rse-r-package.md"}
+```
+
+## Objectives {#rse-r-package-objectives}
+
+```{r, child="objectives/rse-r-package.md"}
+```
+
+## What's in an R package? {#rse-r-package-intro}
+
+> Another response of the wizards,
+> when faced with a new and unique situation,
+> was to look through their libraries to see if it had ever happened before.
+> This was...a good survival trait.
+> It meant that in times of danger you spent the day sitting very quietly in a building with very thick walls.
+>
+> -- Terry Pratchett
+
+The more software you write,
+the more you realize that a programming language is really just
+a way to build and combine software libraries.
```

Comment:
You'll have to be careful here, since Python and R use different terms for a software "bundle" ("library" vs "package"). Maybe either use another term or do something like `libraries/package`? And considering this is on the R side, I would switch everywhere from `library` to `package`.

Reply:
:+1: